docs/reference/rest-api/security/grant-api-keys.asciidoc

```diff
@@ -83,15 +86,28 @@ It supports nested data structure.
 Within the `metadata` object, keys beginning with `_` are reserved for
 system usage.
 
+`client_authentication`::
+(Optional, object) When using the `access_token` grant type, and when supplying a
+JWT, this specifies the client authentication for <<JWT access tokens, jwt-realm-oauth2>>
+ (i.e. what's normally specified by the `ES-Client-Authentication` request header).
+
+`scheme`:::
+(Required, string) The scheme (case-sensitive) as it's supplied in the
+`ES-Client-Authentication` request header.
+
+`value`:::
+(Required, string) The value that follows the scheme for the client credentials
+as it's supplied in the `ES-Client-Authentication` request header.
+
 `grant_type`::
 (Required, string)
 The type of grant. Supported grant types are: `access_token`,`password`.
 
 `access_token`:::
 (Required*, string)
-In this type of grant, you must supply an access token that was created by the
-{es} token service. For more information, see
-<<security-api-get-token>> and <<encrypt-http-communication>>.
+In this type of grant, you must supply either an access token, that was created by the
+{es} token service (see <<security-api-get-token>> and <<encrypt-http-communication>>),
+or a <<JWT, jwt-auth-realm>>.
```

Comment:
> or a <<JWT, jwt-auth-realm>>.

or a <<JWT, jwt-auth-realm>> (either a JWT access_token or a JWT id_token) 
(suggestion to help say id_tokens are supported too)